networking/podenv.go

```diff
@@ -218,6 +222,11 @@ func copyFileToDir(src, dstdir string) (string, error) {
 }
 
 func loadUserNets(localConfig string, netsLoadList common.NetList) ([]activeNet, error) {
+	if netsLoadList.None() {
+		log.Printf("Networking namespace with loopback only")
```

Comment:
TBH, it's not clear to me. the log package seems to break the lines between log entries, but according to https://golang.org/src/log/log.go?s=8595:8639#L278 it's just using fmt.Printf(), so it would seem correct to use "\n".